sn_transfers/src/wallet/mod.rs

```diff
@@ -71,34 +28,56 @@ pub use self::{
 pub(crate) use keys::store_new_keypair;
 
 use crate::{NanoTokens, UniquePubkey};
-use std::collections::{BTreeMap, BTreeSet};
 
-#[derive(Default, serde::Serialize, serde::Deserialize)]
-/// This assumes the CashNotes are stored on disk
+// How large to keep wallet cache size
+pub const WALLET_CACHE_SIZE: usize = 100;
```

Comment:
better make this number to be something like: `std::cmp::max(100, batch_size * 2)`,
in case someone set the BATCH_SIZE to a bigger number say 128 ?

Reply:
Tidied up with the removal of this all together